File: src/app/Models/Domain.cs
public record Artist(int TrackID, string Name);

public record Album(int AlbumID, int ArtistID, string Title);

public record Track(int TrackID, int AlbumID, string Title, string Url);

public record TrackSummary(
    int AlbumArtistID,
    string AlbumArtist,
    int AlbumID,
    string Album,
    int ArtistID,
    string Artist,
    int TrackID,
    int? TrackNumber,
    string Title,
    string Url);

File: src/app/Domain.fs
module jook.Domain

[<CLIMutable>]
type Artist = {
    TrackID: int
    Name: string }

[<CLIMutable>]
type Album = { 
    AlbumID: int
    ArtistID: int
    Title: string }

[<CLIMutable>]
type Track = { 
    TrackID: int
    AlbumID: int
    Title: string
    Url: string }

[<CLIMutable>]
type TrackSummary = {
    AlbumArtistID: int
    AlbumArtist: string
    AlbumID: int
    Album: string
    ArtistID: int
    Artist: string
    TrackID: int
    TrackNumber: int option
    Title: string
    Url: string }
